core/commands/VersionAddCommand.php

```diff
@@ -55,6 +61,9 @@ function validate() {
 		if( is_blank( $t_name ) ) {
 			throw new ClientException( 'Invalid version name', ERROR_EMPTY_FIELD, array( 'name' ) );
 		}
+
+		$t_timestamp = $this->payload( 'timestamp', '' );
+		$this->timestamp = is_blank( $t_timestamp ) ? null : date_strtotime( $t_timestamp );
```

Comment:
Maybe this is the wrong place to deal with date format.
Does this mean that the payload of a REST client must be prepared depending on  configuration option `normal_date_format`?

Reply:
> Does this mean that the payload of a REST client must be prepared depending on configuration option normal_date_format?

No. 

The API will accept dates in _normal_date_format_ **in addition** to the standard formats accepted by strtotime